README.md

```diff
@@ -166,6 +167,11 @@ Enter a PSTN number and press the call button to place a call.
 
 <img height="500px" src="images/quickstart/make_call_to_number.png">
 
+## Examples
+
+In addition to the quickstart, we have added a common use case that show you how to create and customize your application:
```

Comment:
In addition to the quickstart, we've added an example of a common use case to show you how to use Android Connection Service:

- [Voice Connection Service] - Demonstrates how to use [Android Connection Service](https://developer.android.com/reference/android/telecom/ConnectionService.html) with the Android Voice SDK